hbase_sql/src/test/java/com/splicemachine/hbase/CheckTableIT.java

```diff
@@ -266,6 +266,24 @@ public void testIndexExcludeDefaults() throws Exception {
     @Test
     public void negativeTests() throws Exception {
 
+
+        try {
+            spliceClassWatcher.execute(String.format("call syscs_util.check_table('%s', '%s', null, 1, null)", SCHEMA_NAME, A, getResourceDirectory(), A));
```

Comment:
Are theses tests related to this change?

Reply:
Not to validate_backup. It is for check_table